loader/loader.go

```diff
@@ -632,6 +633,19 @@ func (l *Loader) convertEnum(tspec *ast.TypeSpec) (*desc.EnumDescriptorProto, er
 	return enum, nil
 }
 
+// formatTypeForPackage will format the type name for that package. If the
+// package is nil, it will format the type for the current file / package that is
```

Comment:
I'd drop the `file / package` bit in favor of just either option, as that makes the sentence unnecessarily complex to parse. We care about the package name, so I'd just say "current package".

Reply:
Done.